src/System.Private.TypeLoader/src/Internal/Runtime/TypeLoader/TypeLoaderEnvironment.Metadata.cs

```diff
@@ -823,11 +823,38 @@ public struct VirtualResolveDataResult
             ExternalReferencesTable externalReferences = default(ExternalReferencesTable);
             externalReferences.InitializeCommonFixupsTable(module);
 
```

Comment:
A corresponding change to TryGetMethodNameAndSigFromVirtualResolveData is missing from this change.